GOVERNANCE.MD

```diff
@@ -1,6 +1,92 @@
+Section 1. Guiding Principle.
+
+The Community Committee will operate transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to the entire ecosystem.
+
+Section 2. Evolution of Node.js Foundation Governance.
+
+Most large, complex open source communities have both a business and a technical governance model. Node.js Foundation’s leadership contains both a Technical Steering Committee (“TSC”) and Maintainers for major components or subsystems. Node.js Foundation’s business leadership is instantiated in a Board of Directors (the “Board”). Community activities, such as events, fall outside of the realm of code and documentation that the TSC oversees. The champions of these organizations have very different skillsets, talents, and concerns. The community organizations of Node.js have long been important to the growth and vitality of the project and will be recognized as the Community Committee("CC") as represented on the Board by the existing Individual Membership Board Directors.
+
+This Community Committee Charter reflects a formal role and the relevance of the voice of community for the governance of Node.js Foundation. The charter amendment process is for the Community Committee(CC) to propose changes using simple majority of the full CC, the proposed changes being subject to review and approval by the Board. The Board may additionally make amendments to the CC charter at any time, though the Board will not interfere with day-to-day discussions, votes or meetings of the CC.
+
+Section 3. Board’s Role in Setting Node.js Foundation’s Strategic Direction.
+
+The Board will set the overall CC Policy. The policy will describe the overarching scope of the Node.js Foundation initiative, Node.js Foundation’s community vision, direction and expectations in the form of intent. The Board will use the CC as a delegate body for community representation, advisement, and overseeing cultural project implementation, scope and direction while they remain within the scope and direction of the policies as described in the CC Policy document and approved by the Board.
+
+Section 4. Establishment of the Community Committee.
+
+Membership is for 6 months. The group will ask on a regular basis if the expiring members would like to stay on. A member just needs to reply to renew. There is no fixed size of the CC. However, the expected target for Advisers, as defined in Section 10, is between 9 and 12, to ensure adequate coverage of important areas of community expertise, balanced with the ability to make decisions efficiently.
+
+In the case where an individual CC member attends fewer than 20% of the regularly scheduled meetings held within a six month period, and does not communicate with the CC their intention not to attend those
++meetings, the member shall be asked to consider voluntarily resigning their
++membership.
```

Comment:
I'm all for making it stricter as well. If it turns out to be too burdensome, we can always revise it later.